db/update-files.d/u12_alter_lubw_parser.sql

```diff
@@ -0,0 +1,143 @@
+CREATE OR REPLACE FUNCTION station_data.lubw_parse()
```

Comment:
some comments about this function would be useful to understand it later on

Reply:
Added comments on structure in [commit](https://github.com/meggsimum/sauber-sdi-docker/pull/120/commits/715cd123f1563ce15fed1187a3691b2cb0eebf30)